main/src/cgeo/geocaching/export/TrailHistoryExport.java

```diff
@@ -113,7 +113,7 @@ protected File doInBackgroundInternal(final TrailHistoryElement[] trail) {
                 gpx.attribute("", "creator", "c:geo - http://www.cgeo.org/");
                 gpx.attribute(NS_XSI, "schemaLocation", NS_GPX + " " + GPX_SCHEMA);
 
-                    final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+                    final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
```

Comment:
I don't think we need a Locale for `SimpleDateFormat` as we give an explicit format string, but it should not harm either. (Valid for the other occurrences of `SimpleDateFormat` as well.)

Reply:
See the lint message:
https://ci.cgeo.org/job/cgeo%20continuous%20integration/3057/androidLintResult/type.-11914057/file.-1822116914/